plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerLauncher.java

```diff
@@ -84,16 +87,24 @@ public JavaLanguageServerLauncher(
     this.notifyTransmitter = notifyTransmitter;
     this.eventService = eventService;
     this.projectManager = projectManager;
+
+    isStarted = new AtomicBoolean();
```

Comment:
Please use AtomicBoolean(false). While I would expect the default value to be false, I have to check the javadoc on AtomicBoolean to be sure.

Reply:
ok